test/verify/check-users

```diff
@@ -1046,7 +1046,7 @@ class TestAccounts(testlib.MachineCase):
         b.wait_not_present("#password-expiration")
 
         b.logout()
-        self.login_and_go("/users", user="scruffy")
+        self.login_and_go("/users", user="scruffy", superuser=False)
```

Comment:
This should be a separate commit with an explanation. It took you so long to figure out, let's please explain it in the commit message for our future selves :grin:

Reply:
I don't like this change -at all-, and I absolutely veto it.  It might be right for this test, but we're doing this to cover up a bug.  At the same time, I agree that it's not directly related to this particular test scenario.

My proposal: in the same commit that "fixes" this testcase, we introduce a separate test case which tests this.  I'm happy to have it marked as an xfail at first with a link to #20263 and (maybe or maybe not) #20262.

Reply:
I'm happy to add an expected failing test, I assume you mean integration not pytest?